src/CommandingOfficers/Commander.java

```diff
@@ -251,6 +253,7 @@ public void activateAbility(CommanderAbility ability, MapMaster map)
     myActiveAbility = ability;
   }
 
+  /** Do not use this unless you ask the Commander how much energy to input */
```

Comment:
example?

Reply:
`Army`'s listeners for combat and mass damage. They call the functions in `Commander` that assess a charge value to add.

Reply:
I meant to give an example of how to do what the comment says to do. And also what the preferred alternative is. If I, an ignorant code monkey, come across this a year from now, how will I know what to do? All this tells me is "don't use this unless you _really_ know what you're doing."